app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -257,6 +270,41 @@ class Core(private val context: Context) {
         }
     }
 
+    private fun getCookiePolicy(): CookiePolicy {
```

Comment:
I would rename to `geCustomCookiePolicy` and remove the condition 
`if (context.settings().blockCookiesInCustomTrackingProtection)` to avoid any misunderstanding as we are returning  `CookiePolicy.ACCEPT_ALL` for all case when `getCookiePolicy()` is called from non custom context, the cookie policy must depend on the tracking protection policy that we have `strict` or `recommended`.